### DISCUSSION TITLE
How to properly slice units spritesheets?

### DISCUSSION THREAD

**Discussion by NaleVex** [open-duelyst/duelyst#273] [General]
Hey! How can I use sprite sheets from the app/resources/units/ folder? Seems like most of the sprites there are placed chaotically in sprite sheets, I can't properly slice sheets into sprites to get smooth animations, slicing by cell count doesn't work because sprites have different widths and offsets from each other, automatic unity slicing doesn't work well. I'm confused on how to prepare these spritesheets to build animations. Thanks in advance.

**Comment by Shizounu** [comment]
Hey so I was hopping through the discussion on this and noticed this... as I had just completed the tool to do this for me xD 
The animation sheet is ordered in reverse - the most bottom right sprite is the first of a sequence and then all sprites are laid out in order. 
The PList file for each thing in the game provides the info what animation is what. 

If people are interested in my tool for the Unity editor to work with this I could share it :D - It is as of writing incomplete, only parsing generals and units into Unity Animators but I am planning to convert the rest as well
